client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,512 @@
+/**
+ * External dependencies
+ */
+import {
+	applyTestFiltersToPlansList,
+	findPlansKeys,
+	getPlan as getPlanFromKey,
+	getYearlyPlanByMonthly,
+	isMonthly,
+	TERM_MONTHLY,
+	TERM_ANNUALLY,
+} from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getPlan, getPlanBySlug, getPlanRawPrice } from 'calypso/state/plans/selectors';
+
+/**
+ * Internal dependencies
+ */
+import './tabbed-plans-style.scss';
+function TabbedPlans( { currencyCode, onUpgradeClick, planProperties } ) {
+	const translate = useTranslate();
+	const tabList = [ 'Limited', 'Professional' ];
+
+	const featureComparisonData = {
+		Professional: [
+			{
+				featureName: translate( 'Install plugins' ),
+				planOne: { included: false, copy: translate( 'None' ) },
+				planTwo: { included: true, copy: translate( '58,000+ Available' ) },
+				planThree: { included: true, copy: translate( '58,000+ Available' ) },
+			},
+			{
+				featureName: translate( 'Themes available' ),
+				planOne: { included: true, copy: translate( '25' ) },
+				planTwo: { included: true, copy: translate( '8,000 to choose from' ) },
+				planThree: { included: true, copy: translate( '8,000 to choose from' ) },
+			},
+			{
+				featureName: translate( 'SEO' ),
+				planOne: { included: true, copy: translate( 'Simple' ) },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **38** times:
`translate( 'Simple', { context: 'Theme Showcase term name'} )` *ES Score: 13*